index.html

```diff
@@ -16,30 +16,30 @@
         <h1>All awsome books</h1>
       </div>
       <section id="bookSec" class="w-25 mx-auto">
-        <div class="row borderT">
+        <!-- <div class="row borderT">
           <div class="col-6 text-left">
-            <h5 class=" m-2">Book 1 by manuel</h5>
+            <h5 class="m-2">Book 1 by manuel</h5>
           </div>
           <div class="col-6 text-end">
-            <input type="submit" id="Add" value="Remove" class="m-2">
+            <input type="submit" id="" value="Remove" class="m-2">
           </div>
         </div>
         <div class="row borderMW">
           <div class="col-6 text-left">
             <h5 class=" m-2">Book 1 by manuel</h5>
           </div>
           <div class="col-6 text-end">

```

Comment:
- [ ] There are so many commented lines of code in your codebase. Let's clean it up by removing them so that you have a clean and more readable codebase.